pulumi/aws/start_all.sh

```diff
@@ -35,9 +39,29 @@ if ! grep --quiet '^PULUMI_STACK=.*' "${script_dir}/config/environment"; then
   echo "PULUMI_STACK=${PULUMI_STACK}" >> "${script_dir}/config/environment"
 fi
 
+if ! grep --quiet '^AWS_DEFAULT_REGION=.*' "${script_dir}/config/environment"; then
```

Comment:
Pulumi and the AWS CLI also read from the `AWS_REGION` environment variable or the `~/aws/credentials` file in order to get the region. This logic may conflict with that mode of operation.

Reply:
I think that this last push should fix it; we are just using the RC from the command itself to tell us if we need to pull the region out of the config.